src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `currentHash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved. The storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
```

Comment:
Also I don't think that the naming is appropriate. What exactly is "current" here, given that the block can be very old?

Reply:
Yep that makes sense, have applied James suggestion of naming `currentHash` -> `hash` and `fromHash` -> `previousHash` 🙏